docs/howto/add_navigation_hub_items.md

```diff
@@ -114,3 +114,18 @@ A setter method `setRoutePrefix` can be used to define a route prefix used to ma
 If given, a Request's route with the same prefix will match (example: `admin_sectionedit` matches the
 `admin_section` prefix).
 
+## Position
+Order of both, links and zones, can be set. It's implemented in a *Symfony way* as a `priority` parameter of service definition tag:
+
+```yaml
+services:
+    ezsystems.platformui.navigationhub.link.admin.dashboard:
+        # ...
+        tags:
+            - name: ezplatform.ui.link
+              priority: 100
+```
+
+Lower `priority` values will make links appear first, those with the same `priority` value will appear in an order of definition, higher values obviously later.
```

Comment:
Yes, that is my main doubt with that. The standardized `priority` in Symfony is used in reverse order. Using that to order graphical elements feels really odd to me.

Doc note: i would avoid terms such as "obviously". They're very subjective, and don't add much.